app-config.production.yaml

```diff
@@ -61,4 +61,5 @@ catalog:
       target: ./apicurio/demo.yaml  
   providers:
     apicurio:
-      url: ${APICURIO_ENDPOINT}    
\ No newline at end of file
+      # url: ${APICURIO_ENDPOINT}    
+      url: https://apicurio-registry-rhaf-apicurio-registry.apps.dev-eng-ocp4-mas.dev.3sca.net
```

Comment:
@aljesusg any reason to switch from an environment variable to a fixed value here?
Would it be possible to keep the env var to increase the separation of concerns?

Reply:
Yes I wanted to test that works the deployment, this is not going to work until we merge https://github.com/Apicurio/apicurio-operate-first/pull/84 where I am adding this variable

Reply:
I'm checking now the isolated-vm issue

Reply:
merged the relevant PR, let me know if you need any further assistance.